bench-run/src/main/scala/tuples/ArrayOps.scala

```diff
@@ -0,0 +1,54 @@
+package dotty.tools.benchmarks.tuples
```

Comment:
This file should be in `bench-run/src/main/scala/dotty/tools/benchmarks/tuples/ArrayOps.scala`
Same for other benchmarks.

Reply:
Ok ! Then, should I also move `Main.scala` to `bench-run/src/main/scala/dotty/tools/benchmarks` ? I'm asking that because I copied the folder structure from `bench`.

Reply:
Yes